### DISCUSSION TITLE
Make keyboard shortcuts work in the background.

### DISCUSSION THREAD

**Discussion by agvozdezkiy** [rafaelmardojai/blanket#211] [Ideas]
Keyboard shortcuts do not work in the background mode. The sound does not turn off.

**Comment by rafaelmardojai** [comment]
Making keyboard shortcuts work when the app isn't focused isn't possible, but you should be able to use your media play/pause key thanks to the MPRIS interface.

If you don't have dedicated media keys in your keyboard you should be able to configure a global shortcut for that.
